pages/reset-password.js

```diff
@@ -0,0 +1,110 @@
+import API_URL from "../components/globalApiUrl.js";
+import Link from "next/link";
+import Router from "next/router";
+import Layout from "../components/layout.js";
+
+export default class extends React.Component {
+  state = { email: "", pin: "", password: "", confirmPassword:"", resettingPassword: false, errorMessage: "" };
+
+  handleChange = event => {
+    const value = event.target.value,
+      name = event.target.name;
+    this.setState({ [name]: value });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    if(this.state.password !== this.state.confirmPassword){
+        this.setState({errorMessage: "Passwords don't match"});
+    }
+    else if (!this.state.resettingPassword) {
+      // User already got the security question, so now send the response to the server
+      this.setState({ errorMessage: "" });
+      
+      fetch(API_URL + "/passwordReset", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({
+         email: this.state.email,
```

Comment:
shouldn't the token and new password be sent in this request?

Reply:
That would be for `changePassword`. This is resetting from forgetting the password.